src/components/Wallet/Asset.tsx

```diff
@@ -34,16 +44,21 @@ const INITIAL_STATE: State['transaction'] = {
   amount: '',
   publicKey: '',
 };
+
 export default function Asset({ balance }: Props) {
   const [{ amount, publicKey }, setTransaction] =
     useState<State['transaction']>(INITIAL_STATE);
   const { view, handleChangeBoolean } = useBoolean();
   const { getBalanceData } = useLoadAccount();
-  const { secretKey } = useBearStore(({ account }) => ({
-    secretKey: account.secretKey,
-  }));
-  const asset =
-    balance.asset_type === 'native' ? 'Lumens (XLM)' : balance.asset_type;
+  const { handleGetTransactions } = useTransaction();
+  const { secretKey, payment, publicKeySend } = useBearStore(
+    ({ account, payment }) => ({
+      secretKey: account.secretKey,
+      publicKeySend: account.publicKey,
+      payment,
+    }),
+  );
+  const asset = parseAssetTypeNativeToXML(balance.asset_type);
 
   const handleSendTransaction = async (
```

Comment:
What happens if our wallet doesn't use Albedo anymore but xBull/Freighter? We must change the business logic and try to adapt that to xBull/Freighter.